dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -76,11 +77,42 @@ object Datadog {
      * @see [DatadogConfig]
      * @throws IllegalArgumentException if the env name is using illegal characters and your
      * application is in debug mode otherwise returns false and stops initializing the SDK
+     * @deprecated Use the [Datadog.initialize] instead which requires a privacy [Consent] parameter.
+     */
+    @Deprecated(
+        "This method is deprecated and uses the [Consent.GRANTED] " +
+            "flag as a default privacy consent.This means that the SDK will start recording " +
+            "and sending data immediately after initialisation without waiting " +
+            "for the user's privacy consent.",
+        ReplaceWith(
+            expression = "Datadog.initialize(context, Consent.PENDING, config)",
```

Comment:
What does `ReplaceWith()` do? It's for IDE auto-fix? If so, maybe we should lead to `.GRANTED`, no?

Reply:
It is IDE auto - fix yes. I am not sure if we should put GRANTED by default there, the client will definitely put his own value anyway but let's see what @xgouchet says also.

Reply:
I agree with setting it to `PENDING` in the updated version, as it's explicitly written, not a default value. This can be used to show this as the recommended way to initialize the library